string AddressToFind            # red/yellow/blue/orange/BuildingPlace
geometry_msgs/PoseStamped AddressPose
---
uint8 state